sympy/core/relational.py

```diff
@@ -543,6 +543,49 @@ def _eval_simplify(self, ratio, measure, rational, inverse):
                 pass
         return e.canonical
 
+    def subs(self, *args, **kwargs):  # should mirror sympy.core.basic.subs
+        r"""Return a new equation with subs applied to both sides.
+
+        **Examples:**
```

Comment:
```suggestion
        Examples
        ========
```
I think this would be more consistent to the format